docs/netlify.toml

```diff
@@ -62,50 +65,59 @@
 	to = "/contributing/sponsor"
 	status = 301
 
+#####
+##### User Guide
+#####
+
+[[redirects]]
+	from = "/app"
+	to = "/user-guide/overview/data-studio-app.html"
+	status = 301
```

Comment:
@phazonoverload Can you please check whether this redirect makes sense? 

Ref. https://github.com/directus/directus/issues/19024

Reply:
Conflicted as there's a Data Studio section in both the user guide and the main dev docs, but yes - in this case I think if people are hitting /app they're learning about the web app and this makes sense.

Reply:
I agree the redirects makes sense as it is. Most of the paths in /app are now in user guide and that path is the best redirect in that scenario